packages/@uppy/transloadit/src/Assembly.js

```diff
@@ -78,6 +79,12 @@ class TransloaditAssembly extends Emitter {
 
       this.emit('connect')
     })
+
+    socket.on('connect_failed', () => {
+      this._onError(new NetworkError('Transloadit Socket.io connection error'))
+      this.socket = null
+    })
+
```

Comment:
Added handling of socket.io’s `connect_failed`, but should the same be done for WebSocket that we use with Companion @ifedapoolarewaju?

Reply:
I guess we could try Renée's suggestions from the slack conversation?

Reply:
I think that could be another followup PR maybe? the Transloadit plugin itself does not rely on the companion-client `Socket`